client/cypress/integration/event-manager/filters_spec.js

```diff
@@ -5,59 +5,247 @@
 /* global cy */
 /* eslint no-undef: "error" */
 
+const delay = 30000;
+const org =
+  ':nth-child(1)  .Select  .Select-control  .Select-arrow-zone  .Select-arrow';
+const loc =
+  ':nth-child(4) .Select  .Select-control  .Select-arrow-zone  .Select-arrow';
+const spons =
+  ':nth-child(5) .Select  .Select-control  .Select-arrow-zone  .Select-arrow';
+const tag =
+  ':nth-child(6) .Select  .Select-control  .Select-arrow-zone  .Select-arrow';
+const time =
+  ':nth-child(7) .Select  .Select-control  .Select-arrow-zone  .Select-arrow';
+const kw=
+  ':nth-child(8) > .form-control';
+
 describe('Filters Testing', () => {
   context('Test for Filters Route', () => {
     it('Should be on Filters page', () => {
       cy.visit('/events');
       cy.get('h3').should('contain', 'Welcome to Event Management');
     });
   });
-  context('Test for Organization Filters', () => {
-    it('should have an arrow button', () => {
-      cy.get(
-        ':nth-child(1)  .Select  .Select-control  .Select-arrow-zone  .Select-arrow',
-        { timeout: 30000 }
-      ) //Gets the location of the Organisation field with a delay of 30000
-        .eq(0);
-      // .click(); //Clicks the toggle button
+
+  context('Test for Organization Filter', () => {
+    it('should have a clickable arrow button', () => {
+      cy.get(org, { timeout: delay })
+        .eq(0)
+        .dblclick(); //Clicks the toggle button
+      // cy.get('div .Select').should('have.attr', 'is-open');
+    });
+
+    it('should have one item', () => {
+      cy.get(org, { timeout: delay })
+        .eq(0)
+        .click();
+      cy.contains('NES').click();
```

Comment:
select a random first element instead of giving names because we won't have this name always in list. It would have different names for different events